js/main.js

```diff
@@ -0,0 +1,21 @@
+const photoUrl=document.getElementById("photolink")
+const img=document.getElementById("img")
+photoUrl.addEventListener('input',function(event){
+  img.setAttribute('src',event.target)
+});
+
+const form=document.querySelector('form')
+
+form.addEventListener('submit',function(event){
+  event.preventDefault()
+  const input={
+    notes:form.notes,
+    title:form.title,
+    photoUrl:form.PhotoURL
```

Comment:
the values here should be `event.target.elements.{matchingElement}.value` where `{matchingElement}` should be the matching `id` of the form element in your HTML document that you're grabbing from